### DISCUSSION TITLE
Identifying Number of Inserts

### DISCUSSION THREAD

**Discussion by littlekittel** [mozman/ezdxf#844] [Q&A]
Hi all,

I have converted a .dwg file into .dxf and I am trying to export several information of it. One thing is a count of certain blocks within a viewport. I have blocks that are inserted in AutoCAD multiple times in the 2D drawing because they are exactly above each other. I can see this in AutoCAD as it is an attribute of that block. I can't find this information anywhere in ezdxf. I have tried several attributes of the INSERT like mcount and also attributes of the block but I can not find it anywhere.
Does anyone have an idea where this could be found?

Thanks for your help!

**Comment by H2Wavvy** [comment]
You could: 

block_name = entity.dxf.name
quantity = sum(1 for e in msp if e.dxftype() == "INSERT" and e.dxf.name == block_name)

Use this with ezdxf, since INSERT entities or blocks have unique names, it should work, or it works for me. 
Hope this helps!
